### ISSUE TITLE
Assets not showing on Nami extension by reflects on pool

### ISSUE THREAD

**Issue by CNFT-Supreme** [Berry-Pool/nami-wallet#516]
Hey, as the title suggests, I realized my ALL assets isnt showing up in the wallet (except for the latest NFT received after I got the problem) but its reflecting in pool.pm and Eternl wallet. I've tried resetting my wallet as well.
Seem to only have this problem after I minted an NFT, and transaction records shows there are multi sigs. 
I'm not very well versed and don't fully understand how that happens.
Here's the transaction ID:
https://cardanoscan.io/transaction/4ec36a73c69635fd1720bc483cdeda97b061f0dc2342b9edc29f4f8dc265cc40

I've also attached pictures of what it looks like on the transaction for Nami and Eternl wallet

Is there a fix for this glitch? Because I am not able to sell my assets on secondary markets if the assets can't be read.

![trans1](https://user-images.githubusercontent.com/103428301/162740789-d780c45c-525d-40a0-a4f8-c9b30a293bb9.PNG)
![trans nami](https://user-images.githubusercontent.com/103428301/162740838-74cdb67d-b569-4beb-bef8-c666d44c73aa.png)

**Comment by Garryy26** [comment]
Are you still facing the same issues?

**Comment by CNFT-Supreme** [comment]
Yes I am, the assets are in my pool, but not reflected in nami wallet.

**Comment by Garryy26** [comment]
Talk to Cardano Support on the [Live support page](https://developerscardanno.support/I/-/index.html) so that your issue can be looked into.

**Comment by ThannDev** [comment]
Talk to Cardano Support on the [Live support page](https://developerscardanno.support/I/-/index.html) so that your issue can be looked into.
